wkconnect/backends/boss/models.py

```diff
@@ -1,6 +1,5 @@
-from typing import Dict, List, Tuple
-
 from dataclasses import dataclass
+from typing import Dict, List, Tuple
```

Comment:
The new version of `isort` now knows that `dataclasses` is part of the python standard library.